### ISSUE TITLE
"Confirm" and "Cancel" buttons don't work in portrait mode (iOS)

### ISSUE THREAD

**Issue by Aleksandern** [mmazzarolo/react-native-modal-datetime-picker#485]
## Versions

- iOS: 14.0
- react-native-modal-datetime-picker: 9.0.0
- [react-native-community/datetimepicker](https://github.com/react-native-community/datetimepicker): 3.0.4
- react-native: 0.63.3
- react: 16.13.1

## Description

"Confirm" and "Cancel" buttons don't work in portrait mode (not tappable).
![ezgif-2-801985478e51](https://user-images.githubusercontent.com/5274465/96577837-7478d080-12fe-11eb-8b21-499223bdce9c.gif)

code:
```
import React, { useState } from "react";
import { Button, View } from "react-native";
import DateTimePickerModal from "react-native-modal-datetime-picker";

const Example = () => {
  const [isDatePickerVisible, setDatePickerVisibility] = useState(false);

  const showDatePicker = () => {
    setDatePickerVisibility(true);
  };

  const hideDatePicker = () => {
    setDatePickerVisibility(false);
  };

  const handleConfirm = (date) => {
    console.warn("A date has been picked: ", date);
    hideDatePicker();
  };

  return (
    <View>
      <Button title="Show Date Picker" onPress={showDatePicker} />
      <DateTimePickerModal
        isVisible={isDatePickerVisible}
        mode="date"
        onConfirm={handleConfirm}
        onCancel={hideDatePicker}
      />
    </View>
  );
};

export default Example;
```

**Comment by midoalone** [comment]
did you get a solution for this? I am also facing it on iOS 14

**Comment by Aleksandern** [comment]
@midoalone 
No I didn't. I've just posted it.

**Comment by midoalone** [comment]
@Aleksandern I discovered that this issue is related to react native Modal class not this library, I am searching for a solution and hope I find a fix
I think it is related to react native 62.2 version

**Comment by Aleksandern** [comment]
@MichielDeMey 
14.1 update fixed that.

**Comment by Aleksandern** [comment]
It looks like this error is still happening on iOS 14.1

**Comment by mmazzarolo** [comment]
🤔 I'll give it a try this weekend. @midoalone do you have any links on the subject? 
Thanks for starting the discussion btw

**Comment by mmazzarolo** [comment]
Working fine in the example, using:
-  "react": "16.13.1",
-  "react-native": "0.63.2",

iPhone 11 - 14.0

![2020-10-23 19 49 33](https://user-images.githubusercontent.com/9536354/97037000-f66c3200-1568-11eb-8aaa-9cb8fa7427bf.gif)

**Comment by czegarram** [comment]
I'm experiencing the same issue in ios 14.

React: 16.9.0
React Native: 0.63.3


![Simulator Screen Shot - iPhone 11 - 2020-10-24 at 18 06 46](https://user-images.githubusercontent.com/4095996/97093680-7e634200-1624-11eb-9943-08bb11cef026.png)

**Comment by czegarram** [comment]
I also tried with:

"react": "16.13.1",
"react-native": "0.63.2",

but no luck.

**Comment by mmazzarolo** [comment]
Hmmm does it work with just the example? Can you provide a MINIMAL repro
repo?

**Comment by mmazzarolo** [comment]
Can't debug this without a minimal reproducible code. Also, the example is working fine for my tests, so I'm closing the issue.

**Comment by OlegPanfyorov** [comment]
Faced exactly the same issue :((((

**Comment by dylan-westbury** [comment]
Facing this issue also. Can't click the confirm or cancel buttons, nothing happens. 

    "react-native-modal-datetime-picker": "^9.1.0",
    "@react-native-community/datetimepicker": "^3.0.9",
    "react-native": "0.63.3",
    "react": "16.13.1",

**Comment by dylan-westbury** [comment]
I tried upgrading to react-native 63.4 and not fixed.

@mmazzarolo I have created a reproducable repo 

https://github.com/dylan-westbury/datepicker-reproduce-confirm-cancel-bug

Running on ios 14.3 iphone 11

**Comment by mmazzarolo** [comment]
@dylan-westbury thank you. Is the example working for you?

**Comment by dylan-westbury** [comment]
@mmazzarolo no can't press cancel or confirm in portrait mode

**Comment by mmazzarolo** [comment]
@dylan-westbury 

Uploading Screen Recording 2021-01-10 at 18.00.20.mov…

**Comment by binlaniua** [comment]
why closed ? ios 14.3 has same issue

**Comment by PentamapRainer** [comment]
Any solution on this behavior?  

https://user-images.githubusercontent.com/6825724/120190032-59692a00-c218-11eb-896a-75f8902602d6.mov

When opening the DatePicker in landscape mode, the view changes to portrait mode. When closing the DatePicker, the view changes to landscape mode again.
